DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml

```diff
@@ -2,59 +2,73 @@
 <div class="content-container">
     <div class="filter-form">
         @{ Html.EnableClientValidation(false); }
-          <form action="@Model.Section.PreviousPageUrl" method="post" id="backForm">
-                 <div id="BackLink" class="breadcrumbs">
-                        <ol>
-                            <li>
-                                <button type="submit" class="button-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</button>
-                            </li>
-                        </ol>
-                </div>
-              @Html.HiddenFor(m => Model.Section.Name)
-              @Html.HiddenFor(m => Model.Section.SectionDataType)
-              @Html.HiddenFor(m => Model.OptionsSelected)
-              @Html.HiddenFor(m => Model.Section.PageNumber)
-              @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
+        <form action="@Model.Section.PreviousPageUrl" method="post" id="backForm">
+            <div id="BackLink" class="govuk-breadcrumbs">
+                <a href="@Model.Section.PreviousPageUrl" class="govuk-back-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</a>
```

Comment:
It will have to be a button, as the form needs to be submitted to preserve state.

Reply:
Changed to button for now and Joe was looking into the style change for it and will be added in the next release